tests/core/test_lightning_optimizer.py

```diff
@@ -206,6 +212,35 @@ def test_state(tmpdir):
     assert optimizer.state == lightning_optimizer.state
 
 
+class TestLightningOptimizerModel(BoringModel):
+
+    def configure_optimizers(self):
+        optimizer = torch.optim.Adam(self.parameters(), lr=0.1)
+
+        optimizer = LARSWrapper(optimizer)
+        return [optimizer]
+
+
+@pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
+@pytest.mark.skipif(not BOLTS_AVAILABLE, reason="Bolts is required for this test")
```

Comment:
since bolts is not a dependency now, will this test ever run?

Reply:
good catch ! Let's me add it back.